src/api/publicApiClient.ts

```diff
@@ -24,7 +24,7 @@ const getUpcomingEvents = (): Promise<EventProps[]> => {
 
 const getAnnouncements = (): Promise<Announcement[]> => {
   return AppAxiosInstance.get(PublicApiClientRoutes.ANNOUNCEMENTS).then(
-    (response) => response.data?.announcements,
+    (response) => response.data.announcements,
```

Comment:
👍

Reply:
Was that question mark signifying that the data field is optional?